devops/jobs/StatusEdxOrg.groovy

```diff
@@ -0,0 +1,97 @@
+/*
+ update the status.edx.org website
+
+ Variables without defaults are marked (required) 
+ 
+ Variables consumed for this job:
+    * SECURE_GIT_CREDENTIALS: secure-bot-user (required)
+    * NOTIFY_ON_FAILURE: alert@example.com
+    * FOLER_NAME: folder
+    * STATUS_REPO: default is git@github.com:edX/status.edx.org
+    * STATUS_BRANCH: default is master
+*/
+
+package devops.jobs
+import static org.edx.jenkins.dsl.Constants.common_logrotator
+import static org.edx.jenkins.dsl.Constants.common_wrappers
+
+
+class StatusEdxOrg {
+
+    public static def job = { dslFactory, extraVars ->
+        dslFactory.job(extraVars.get("FOLDER_NAME","Monitoring") + "/status-edx-org") {
+               
+            logRotator common_logrotator
+            wrappers common_wrappers
+
+            wrappers{
+                credentialsBinding{
+                    file('AWS_CONFIG_FILE','tools-edx-jenkins-aws-credentials')
+                    string('ROLE_ARN', 'tools-jenkins-to-status-edx-org-role-arn')
+                }
+            }
+
+            def gitCredentialId = extraVars.get('SECURE_GIT_CREDENTIALS','')
```

Comment:
Can we make a deploy key for this rather than using edx-secure?
https://developer.github.com/v3/guides/managing-deploy-keys/#deploy-keys
You generate an ssh key, put it in lastpass, give github the public part for the edx.status.org repo (we can make sure you have access to do that) and then give Jenkins the private part as a folder level credential.